src/components/core/DatePicker/DatePicker.recipe.ts

```diff
@@ -0,0 +1,97 @@
+import { datePickerAnatomy } from "@ark-ui/anatomy";
+import { defineSlotRecipe } from "@pandacss/dev";
+
+/**
+ * Date picker style recipe.
+ */
```

Comment:
probably should add cursor `pointer` for `clearTrigger` in this recipe.

Reply:
I thought about that too, will add!